mailpoet/tests/integration/Doctrine/EventListeners/ValidationTest.php

```
@@ -47,7 +47,7 @@ public function testItValidatesNewEntity() {
     try {
       $this->entityManager->flush();
       $this->fail('Validation exception was not thrown.');
-    } catch (ValidationException $e) {
+    } catch (ValidationException $e) { // @phpstan-ignore catch.neverThrown (thrown from a Doctrine preFlush event listener; PHPStan can't trace event-driven throws)
```

PHPStan (2.1.51)

[error] No error with identifier catch.neverThrown is reported on line 50.


(ignore.unmatchedIdentifier)